web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -1289,10 +1287,9 @@
               });
             } else {
             	var olL = getTheLayerFromMap(map, name, url);
-            	if(olL && md && md['geonet:info']['uuid']) {
-                olL.set('MDuuid', md['geonet:info']['uuid']);
-                olL.set('metadataUuid', md['geonet:info']['uuid']);
-              }
+                if(olL && md && md.getUuid()) {
```

Comment:
i would better attach the `md` to the layer
`olL.set('md', md)` then you'll get the uuid from the `layer.get('md')`.